mix.exs

```diff
@@ -31,9 +31,8 @@ defmodule Samly.Mixfile do
     [
       {:plug, "~> 1.6"},
       {:esaml, "~> 4.2"},
-      {:sweet_xml, "~> 0.6.6"},
-      {:ex_doc, "~> 0.19.0", only: :dev, runtime: false},
-      {:inch_ex, "~> 1.0", only: [:dev, :test]}
```

Comment:
Not a fan of `mix inch`? I've not seen or used it before but 🤷

Reply:
Oh, I like the idea, but it doesn't seem to work on 1.12 😕